### DISCUSSION TITLE
If I use the "Reset Windows" feature in Windows 11, will my Scoop apps be preserved?

### DISCUSSION THREAD

**Discussion by TechPro424** [ScoopInstaller/Scoop#6164] [Help and Questions]
If I use the "Reset Windows" feature in Windows 11, will my Scoop apps be preserved?

**Comment by HUMORCE** [comment]
You need to edit your PATH environment variable even Scoop exists after reset.

For default installation of Scoop(which it is install to $HOME), if you choose **keep my files** when resetting, it will not be deleted.

**Everything is based on Microsoft's documentation, I have never used Reset Windows**

- `scoop export`
- backup Scoop manually, i.e.: ( copy Scoop dir to another drive, then replace it to new installation
